planetiler-custommap/src/main/resources/samples/shortbread.yml

```diff
@@ -344,7 +346,6 @@ layers:
           - sand
         13:
           amenity: grave_yard
-          natural: wood
```

Comment:
Would be interesting to throw an error for this.
I don't think overrides should be able to overlap.